### ISSUE TITLE
Draft of NDBP Website (Desktop)

### ISSUE THREAD

**Issue by tylowe-labs** [tylowe-labs/MeinVokabular#1]
We have already done...
- Lo-Fidelity Wireframes (understand basic architecture + data required)
- Discussed the minimum set of features for the NDBP website

To complete a draft, the website needs the following sections:
1. Home Page -> High-level overview of the NDBP
2. About Section -> About Us, Board, Team, Partners, Fin Statements
3. Products Section -> Housing, Commerical, Solar Loans, Apply
4. New & Announcements Section -> News, Notices of Sales, Procurements, Jobs
6. Contact Us Page -> Form + Location

There are also several data models that will hold dynamic website information. These data models will create the webpages described above and *also* have web forms for NDBP staff to easily update w/out needing WIX:
1. Board of Directors
    - Name (string)
    - Title (string)
    - Start Date (date)
    - Description (string)
    - Image (media)
2. Staff
    - Name (string)
    - Title (string)
    - Business Unit (Manager, Finance, Housing, Commercial)
    - Description (string)
    - Start Date (date)
    - Image (media)
3. Housing Loans
    - Name (string)
    - Description (string)
    - Interest Rates (string)
    - Fees (string)
5. Commercial Loans
    - Name (string)
    - Description (string)
    - Interest Rates (string)
    - Fees (string)
7. Solar Loans
    - Name (string)
    - Description (string)
    - Interest Rates (string)
    - Fees (string)
9. News
    - Title (string)
    - Publish date (date)
    - Cover Photo (media)
    - Body (markdown string)
    - Images (list of media, optional, for carousel @ end) 
11. Notices of Sales
    - Title (string)
    - Publish Date (date)
    - Cover Photo (media)
    - Body (markdown string)
    - Images (list media, optional, for carousel @ end)
13. Procurements
    - Title (string)
    - Publish Date (date)
    - Body (markdown string)
    - RFP File (media)
15. Careers 
    - Title (string)
    - Publish Date (date)
    - Body (markdown string) 
    - Active (boolean)
17. Financial Statements     
   - Fiscal Year (number)    
   - Financial Statements (media)

Once this is complete, we can begin moving towards production: elicit feedback + real text and data to insert into website, refactor website to update design as necessary, WIX purchases
